components/portal/main/partials/header.html

```diff
@@ -73,18 +73,10 @@ <h1 class="md-title" layout="row">
 
     <!-- SEARCH BUTTON AND USERNAME MENU XS-ONLY -->
     <div layout="row" layout-align="end center" flex-xs="60" class="top-bar-controls-xs" hide-gt-xs>
-      <md-button class="md-icon-button" aria-label="Toggle {{portal.theme.title}} search" ng-click="toggleSearch()">
-        <md-icon ng-if="!searchExpanded">search</md-icon>
-        <md-icon ng-if="searchExpanded">close</md-icon>
-        <md-tooltip class="top-bar-tooltip" md-direction="bottom" md-delay="500">
-          <span ng-if="!searchExpanded">Search</span>
-          <span ng-if="searchExpanded">Close search</span>
-        </md-tooltip>
-      </md-button>
+      <search ng-show="APP_FLAGS.showSearch" mode="mobile-search" layout="row" layout-align="end center" hide-gt-xs></search>
```

Comment:
Since we don't expect `APP_FLAGS.showSearch` to change live, would it be any better for this to use `ng-if` rather than `ng-show`?

Reply:
I might be slightly better since it wouldn't have to initialize the controller. Switched it.